assets/javascript/gallery.js

```diff
@@ -0,0 +1,10 @@
+$(function(){
```

Comment:
the titles and descriptions of the albums, in the json file, are named in some odd way that go couldn't access. I've spent some time trying to make it server side but it didn't work. JavaScript can access these fields and, because I had to get the json file with an ajax request, I had to use jquery.